docs/APIs/communication/network_sockets.md

```diff
@@ -30,11 +30,11 @@ Existing network interfaces:
 
 ## The SocketAddress class
 
-Use the [SocketAddress](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/classSocketAddress.html) class to represent the IP address and port pair of a unique network endpoint. Most network functions are also overloaded to accept string representations of IP addresses, but SocketAddress can be used to avoid the overhead of parsing IP addresses during repeated network transactions, and can be passed around as a first class object.
+Use the [SocketAddress](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.3/api/classSocketAddress.html) class to represent the IP address and port of a network endpoint. Most network functions are also overloaded to accept string representations of IP addresses, but SocketAddress can be used to avoid the overhead of parsing IP addresses during repeated network transactions, and can be passed around as a first class object.
 
 ## Network errors
 
-The convention of the network-socket API is for functions to return negative error codes to indicate failure. On success, a function may return zero or a non-negative integer to indicate the size of a transaction. On failure, a function must return a negative integer, which should be one of the error codes in the `nsapi_error_t` enum ([here](https://github.com/mbedmicro/mbed/blob/master/features/net/network-socket/nsapi_types.h#L27)):
+The convention of the network-socket API is for functions to return negative error codes to indicate failure. On success, a function may return zero or a non-negative integer to indicate the size of a transaction. On failure, a function must return a negative integer, which should be one of the error codes in the `nsapi_error_t` enum ([here](https://github.com/ARMmbed/mbed-os/blob/master/features/netsocket/nsapi_types.h)):
```

Comment:
We're asking about using the link to the rendered doxy on our docs site rather than sending to GitHub